Composer/packages/client/src/recoilModel/dispatchers/project.ts

```diff
@@ -484,11 +483,8 @@ export const projectDispatcher = () => {
             if (settingStorage.get(projectId)) {
               settingStorage.remove(projectId);
             }
-            const currentBotProjectFileIndexed: BotProjectFile = botFiles.botProjectSpaceFiles[0];
-            callbackHelpers.set(botProjectFileState(projectId), currentBotProjectFileIndexed);
 
-            const mainDialog = await initBotState(callbackHelpers, projectData, botFiles);
-            callbackHelpers.set(botProjectIdsState, [projectId]);
```

Comment:
Question (not blocking, just curious) - why are these lines being deleted?

Reply:
this happens inside the call to openRootBotandSkill